src/styles/add/header.scss

```
@@ -0,0 +1,56 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding-inline: 50px;
+  background-color: $fourth-color;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  font-size: 12px;
+  font-weight: 500;
+
+  &__img {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav__items {
```

if `nav` is a separate BEM block it should be in a separate file, not in `header.scss`

```suggestion
.nav {
  &__items {
```